components/script/layout_interface.rs

```diff
@@ -92,6 +92,19 @@ pub enum ReflowQueryType {
     ContentBoxesQuery(TrustedNodeAddress),
 }
 
+/// Extra information concerning the reason for reflowing.
```

Comment:
Those 3 enums are required to call flow(). Wouldn't make sense to keep them grouped together instead of in 3 locations?